tools/tscdocgen/main.go

```diff
@@ -170,6 +170,7 @@ var gitHubBaseURLs = map[string]string{
 	"@pulumi/azuread":      "https://github.com/pulumi/pulumi-azuread/blob/{githash}/sdk/nodejs",
 	"@pulumi/cloud":        "https://github.com/pulumi/pulumi-cloud/blob/{githash}/api",
 	"@pulumi/cloudflare":   "https://github.com/pulumi/pulumi-cloudflare/blob/{githash}/sdk/nodejs",
+	"@pulumi/datadog":		"https://github.com/pulumi/pulumi-datadog/blob/{githash}/sdk/nodejs",
```

Comment:
Nit: Indentation in the middle looks off.

Assuming I counted the spaces correctly:

```suggestion
	"@pulumi/datadog":    "https://github.com/pulumi/pulumi-datadog/blob/{githash}/sdk/nodejs",
```